administration/jsondb.md

```diff
@@ -30,3 +30,154 @@ Example file -:
 ```
 
 ```
+
+----
+
+openHAB stores configuration information in JSON (JavaScript Object Notation) formatted (structured) text files located in the `OPENHAB_USERDATA/jsondb/` directory.
+
+## Storage Scope
+
+All configuration information regarding _**Items, Links, and Things**_ which are defined via the User Interfaces (Paper UI, HABmin, REST) or via internal openHAB services.
+
+Note: The JSON DB does NOT store information for manually configured Items, Links, or Things, since these are already stored in files within the `OPENHAB_CONF` sub-directories (e.g. `/etc/openhab2/items/`).
+
+## Storage Purpose
+
+The JSON DB Storage can be used for:
```

Comment:
Consider eliminating the word "The" at the beginning of, "The JSON DB Storage..."